packages/scandipwa/src/component/ProductPrice/ProductPrice.component.js

```diff
@@ -78,9 +78,9 @@ export class ProductPrice extends PureComponent {
         const { mix } = this.props;
 
         return (
-            <p block="ProductPrice" aria-label="Product Price" mix={ mix }>
+            <h3 block="ProductPrice" aria-label="Product Price" mix={ mix }>
```

Comment:
So while products will be loading, we will have +20 h3 tags? I don't think it semantically it is correct to set h3 for price or its placeholder.